src/main/scala/temple/ast/Templefile.scala

```diff
@@ -19,7 +20,24 @@ case class Templefile(
   def servicesWithPorts: Iterable[(String, ServiceBlock, Ports)] =
     services
       .zip(Iterator.from(1025, step = 2)) // 1024 is reserved for auth service
-      .map { case ((name, service), port) => (name, service, Ports(port, port + 1)) }
+      .map {
+        case ((name, service), port) =>
+          if (name == "Auth") (name, service, Ports(ProjectConfig.authPort, ProjectConfig.authPort + 1))
```

Comment:
Hmm I'm not really a fan of this string comparison, especially given that there's no `.lowercase` or such. How about creating an `AuthServiceBlock` subclass and then you match on the types?

Reply:
Also, how about bringing `ProjectConfig.authPort + 1` into its own variable in `ProjectConfig`?